notebooks/neural_networks/lstm.py

```diff
@@ -68,24 +104,31 @@ def build_model(input_size, hidden_size, output_size):
 optimizer.set_train_summary(train_summary)
 optimizer.set_val_summary(val_summary)
 print "saving logs to ",app_name
-# Boot training process
-trained_model = optimizer.optimize()
-print "Optimization Done."
+
+
+# In[6]:
+
+get_ipython().run_cell_magic(u'time', u'', u'# Boot training process\ntrained_model = optimizer.optimize()\nprint "Optimization Done."')
+
+
+# In[7]:
```

Comment:
I use the following command to generate python code:
`jupyter nbconvert rnn.ipynb --to script`

It seems because `%%time` this line in some Jupyter cells, so the corresponding code can not be generated correctly.